tests/Unit.Tests/Utils/MarkdownFormatter.Tests.cs

```diff
@@ -126,46 +126,46 @@ public void NoHtml()
             Assert.That(GetHtml(markdown), Is.EqualTo(expected));
         }
 
-        [Test]
-        public void Bold1()
-        {
-            var markdown = "**booyah**";
-            var expected = "<p class=\"govuk-body\"><span class=\"govuk-!-font-weight-bold\">booyah</span></p>";
-            Assert.That(GetHtml(markdown), Is.EqualTo(expected));
-        }
-
-        [Test]
-        public void Bold2()
-        {
-            var markdown = "- **booyah**";
-            var expected = "<ul class=\"govuk-list\">\n<li><span class=\"govuk-!-font-weight-bold\">booyah</span></li>\n</ul>";
-            Assert.That(GetHtml(markdown), Is.EqualTo(expected));
-        }
-
-        [Test]
-        public void Bold3()
-        {
-            var markdown = "*****";
-            var expected = "<p class=\"govuk-body\"><span class=\"govuk-!-font-weight-bold\">*</span></p>";
-            Assert.That(GetHtml(markdown), Is.EqualTo(expected));
-        }
-
-        [Test]
-        public void NotBold1()
-        {
-            var markdown = @"**boo
-yah**";
-            var expected = "<p class=\"govuk-body\">**boo</p>\n<p class=\"govuk-body\">yah**</p>";
-            Assert.That(GetHtml(markdown), Is.EqualTo(expected));
-        }
-
-        [Test]
-        public void NotBold2()
-        {
-            var markdown = @"****";
-            var expected = "<p class=\"govuk-body\">****</p>";
-            Assert.That(GetHtml(markdown), Is.EqualTo(expected));
-        }
+//         [Test]
```

Comment:
Do we need to keep commented out code? Can we delete, please?

Reply:
Done - thanks :+1: